src/job/jobHandler.ts

```diff
@@ -369,6 +369,7 @@ export abstract class JobHandler {
       GATSBY_BASE_URL: this._config.get<string>('gatsbyBaseUrl'),
       PREVIEW_BUILD_ENABLED: this._config.get<string>('previewBuildEnabled'),
       GATSBY_TEST_SEARCH_UI: this._config.get<string>('featureFlagSearchUI'),
+      GATSBY_SHOW_CHATBOT: this._config.get<string>('gatsbyUseChatbot'),
```

Comment:
I know this PR came before the Slack discussion last Friday, but would you like to use the regular env variables as opposed to using the node config library?

Reply:
i think we can leave this as a conversion across all env vars instead of a mix usage. WDYT?